Test -> Main Merge, all tested

Ethodeus opened the pull request.

brianschnee commented:
Could you please include details about this version and a collection of the features we are implementing. Thank you

Ethodeus commented:
Version 4

Features:

- Added media queries to make the website responsive in all viewports.
- Added descriptions to each resource object and updated how those objects are rendered to the DOM. 
- Changed the on hover animation on the search bar to be on focus instead. 
- Updated contributing details and guidelines. 
- Updated project description. 

File changes:

resouces.js 
> Added description to every resource object. 

index.ejs
> Added descriptions to default render.

main.js 
> Added descriptions to keyword search render

styles.css
> Added media queries to support all viewports. 

> Changed search bar animation on desktop.

brianschnee commented:
Perfect, thank you so much!
